src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -694,13 +705,17 @@ private synchronized void regenerateAITasks() {
 		pane8.setOpaque(false);
 		pane8.add(layerList);
 
-		JPanel spo6 = new JPanel(new GridLayout(4, 2, 2, 2));
+		JPanel spo6 = new JPanel(new GridLayout(5, 2, 2, 2));
 		spo6.setOpaque(false);
 
 		spo6.add(HelpUtils.wrapWithHelpButton(this.withEntry("entity/sound"),
 				L10N.label("elementgui.living_entity.sound")));
 		spo6.add(livingSound);
 
+		spo6.add(HelpUtils.wrapWithHelpButton(this.withEntry("entity/celebration_sound"),

```

Comment:
Given this is the most specific one, it should be placed at the bottom, not somewhere in between

Reply:
Fixed